### PR TITLE
Moved runtime Dockerfile to this project

### PR DESCRIPTION
**Pull request by berndruecker**
related to https://github.com/camunda/connector-sdk/issues/195

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -35,9 +35,16 @@ jobs:
...
+        push: ${{ github.ref == 'refs/heads/main' }}
+        tags: camunda/connectors:SNAPSHOT
+
     - name: Build and Push Docker Image tag SNAPSHOT
       uses: docker/build-push-action@v3
       with:
-        context: .
+        context: bundle/
         push: ${{ github.ref == 'refs/heads/main' }}
-        tags: camunda/connectors-bundle:SNAPSHOT
```
Comment (tmetzke) [inline .github/workflows/CI.yml:43]: Should this really change? I think this needs to stay as is?

File: runtime/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+ARG SPRING_ZEEBE_VERSION=8.1.4
+FROM eclipse-temurin:17.0.5_8-jre
+
+RUN mkdir /opt/app
+# Download connectors from maven central
+ADD https://repo1.maven.org/maven2/io/camunda/spring-zeebe-connector-runtime/${AWS_LAMBDA_VERSION}/spring-zeebe-connector-runtime-${SPRING_ZEEBE_VERSION}.jar /opt/app/
```
Comment (tmetzke) [inline runtime/Dockerfile:6]:
```suggestion
ADD https://repo1.maven.org/maven2/io/camunda/spring-zeebe-connector-runtime/${SPRING_ZEEBE_VERSION}/spring-zeebe-connector-runtime-${SPRING_ZEEBE_VERSION}.jar /opt/app/
```

Comment (berndruecker) [reply runtime/Dockerfile:6]: it does not have the `with-depencies` suffix - there is only one jar built in the runtime module. The version makes a lot of sense though 🙈

Comment (tmetzke) [reply runtime/Dockerfile:6]: But the Connector runtime JAR we create should have all dependencies included using Maven Shade 🤔

Comment (berndruecker) [reply runtime/Dockerfile:6]: Let me look into this

Comment (berndruecker) [reply runtime/Dockerfile:6]: grafik

The shaded version has the normal artifact name

Comment (tmetzke) [reply runtime/Dockerfile:6]: https://github.com/camunda-community-hub/spring-zeebe/pull/260

Comment (tmetzke) [reply runtime/Dockerfile]: Why do I not get a shaded artifact? 🤔

Comment (tmetzke) [reply runtime/Dockerfile]: What do you run to get those artifacts?

Comment (berndruecker) [reply runtime/Dockerfile]: Use the latest master version - there was something forgotten ;-)
https://github.com/camunda-community-hub/spring-zeebe/blob/master/connector-runtime/pom.xml#L100

**Comment by berndruecker** [comment]
@tmetzke Looks good now and works with spring-zeebe 8.1.3 - shall we merge it now?

File: .github/workflows/CI.yml [hunk 3]
```diff
@@ -35,9 +42,16 @@ jobs:
...
         password: ${{ steps.secrets.outputs.DOCKERHUB_PASSWORD }}
 
+    - name: Build and Push Runtime Docker Image tag SNAPSHOT
+      uses: docker/build-push-action@v3
+      with:
+        context: runtime/
+        push: ${{ github.ref == 'refs/heads/main' }}
+        tags: camunda/connectors:SNAPSHOT
+
     - name: Build and Push Docker Image tag SNAPSHOT
```
Comment (tmetzke) [inline .github/workflows/CI.yml:52]:
```suggestion
    - name: Build and Push Bundle Docker Image tag SNAPSHOT
```

**Review by tmetzke — APPROVED** [review]

**Comment by tmetzke** [comment]
Thanks, @berndruecker, really cool 👍

**Pull request merged**